service/controller/clusterapi/v30/cloudconfig/testdata/case-0-tccp-test.golden

```diff
@@ -805,7 +805,7 @@
tup-kubelet-config.service\nAfter=k8s-setup-network-env.service k8s-setup-kubelet-config.service\nDescription=k8s-kubelet\nStartLimitIntervalSec=0\n[Service]\nTimeoutStartSec=300\nRestart=always\nRestartSec=0\nTimeoutStopSec=10\nCPUAccounting=true\nMemoryAccounting=true\nSlice=kubereserved.slice\nEnvironmentFile=/etc/network-environment\nEnvironment=\"IMAGE=quay.io/giantswarm/hyperkube:v1.14.6\"\nEnvironment=\"NAME=%p.service\"\nEnvironment=\"PATH=/opt/bin/:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin\"\nExecStartPre=/usr/bin/docker pull $IMAGE\nExecStartPre=-/usr/bin/docker stop -t 10 $NAME\nExecStartPre=-/usr/bin/docker rm -f $NAME\nExecStart=/bin/sh -c \"/usr/bin/docker run --rm --pid=host --net=host --privileged=true \\\n-v /:/rootfs:ro,rshared \\\n-v /sys:/sys:ro \\\n-v /dev:/dev:rw \\\n-v /var/log:/var/log:rw \\\n-v /run/calico/:/run/calico/:rw \\\n-v /run/docker/:/run/docker/:rw \\\n-v /run/docker.sock:/run/docker.sock:rw \\\n-v /usr/bin/docker:/usr/bin/docker \\\n-v /run/metadata/torcx:/run/metadata/torcx \\\n-v /run/torcx/:/run/torcx/ \\\n-v /usr/lib/os-release:/etc/os-release \\\n-v /usr/share/ca-certificates/:/etc/ssl/certs \\\n-v /var/lib/calico/:/var/lib/calico \\\n-v /var/lib/docker/:/var/lib/docker:rw,rshared \\\n-v /var/lib/kubelet/:/var/lib/kubelet:rw,rshared \\\n-v /etc/kubernetes/ssl/:/etc/kubernetes/ssl/ \\\n-v /etc/kubernetes/config/:/etc/kubernetes/config/ \\\n-v /etc/kubernetes/kubeconfig/:/etc/kubernetes/kubeconfig/ \\\n-v /etc/kubernetes/manifests/:/etc/kubernetes/manifests/ \\\n-v /etc/cni/net.d/:/etc/cni/net.d/ \\\n-v /opt/cni/bin/:/opt/cni/bin/ \\\n-v /opt/bin:/opt/bin \\\n-v /usr/sbin/iscsiadm:/usr/sbin/iscsiadm \\\n-v /etc/iscsi/:/etc/iscsi/ \\\n-v /dev/disk/by-path/:/dev/disk/by-path/ \\\n-v /dev/mapper/:/dev/mapper/ \\\n-v /lib/modules:/lib/modules \\\n-v /usr/sbin/mkfs.xfs:/usr/sbin/mkfs.xfs \\\n-v /usr/lib64/libxfs.so.0:/usr/lib/libxfs.so.0 \\\n-v /usr/lib64/libxcmd.so.0:/usr/lib/libxcmd.so.0 \\\n-v /usr/lib64/libreadline.so.7:/usr/lib/libreadline.so.7 \\\n-e ETCD_CA_CERT_FILE=/etc/kubernetes/ssl/etcd/server-ca.pem \\\n-e ETCD_CERT_FILE=/etc/kubernetes/ssl/etcd/server-crt.pem \\\n-e ETCD_KEY_FILE=/etc/kubernetes/ssl/etcd/server-key.pem \\\n-e PATH \\\n--name $NAME \\\n$IMAGE \\\n/hyperkube kubelet \\\n--node-ip=${DEFAULT_IPV4} \\\n--config=/etc/kubernetes/config/kubelet.yaml \\\n--containerized \\\n--enable-server \\\n--logtostderr=true \\\n--cloud-provider=aws \\\n--image-pull-progress-deadline=1m \\\n--network-plugin=cni \\\n--register-node=true \\\n--register-with-taints=node-role.kubernetes.io/master=:NoSchedule \\\n--feature-gates=TTLAfterFinished=true \\\n--kubeconfig=/etc/kubernetes/kubeconfig/kubelet.yaml \\\n--node-labels=\"node.kubernetes.io/master,node-role.kubernetes.io/master,kubernetes.io/role=master,role=master,ip=${DEFAULT_IPV4},k1=v1,k2=v2\" \\\n--v=2\"\nExecStop=-/usr/bin/docker stop -t 10 $NAME\nExecStopPost=-/usr/bin/docker rm -f $NAME\n[Install]\nWantedBy=multi-user.target\n",
```

Comment:
The test now only injects the string we see in the golden file. Before the key func result was in there because we did not configure anything but only used the one key func. This is dynamic now and defined by the test.